### PR TITLE
add search functionlity

### PR DESCRIPTION
**Pull request by cpreston321**
Added search to website to make it easier to filter. 

Here is a video !

https://user-images.githubusercontent.com/6619884/204852185-35fae74a-f6d0-4699-b40c-194b128ff50a.mov

### CODE REVIEW

**Comment by cpreston321** [comment]
@pi0 if you want me to add or change the functionality let me know. I tried to make it dead simple 

💚

File: pages/index.vue [hunk 1]
```diff
@@ -1,11 +1,23 @@
 <template>
-  <div class="project-list">
-    <ProjectCard v-for="repo in repos" :key="repo.id" :repo="repo" />
+  <div>
+    <div class="project-input__wrapper">
+      <search-logo class="project-input-logo" />
```
Comment (manniL) [inline pages/index.vue:4]:
```suggestion
      <SearchLogo class="project-input-logo" />
```
For consistency

**Review by manniL — APPROVED** [review]
LGTM!

**Review by pi0 — APPROVED** [review]
LGTM. It would be nice to followup:

- Create Search component
- Support search on description and keywords as well 
- (optionally) allow fuzzy search

**Pull request merged**